File: Kms.Security.Portal/app/nodak/common/enterprisePositionPost/enterprisePositionPost.dto.ts
module nodak.common.models {

    export class EnterprisePositionPostDto{
        private id: any
        set Id(value)
        { this.id = value; }
        get Id()
        { return this.id; }

        private title: string;
        get Title() {
            return this.title;
        }
        set Title(value) {
            this.title = value;
        }

        private description: string;
        get Description() {
            return this.description;
        }
        set Description(value) {
            this.description = value;
        }

        private enterprisePositionId: string;
        get EnterprisePositionId() {
            return this.enterprisePositionId;
        }
        set EnterprisePositionId(value) {
            this.enterprisePositionId = value;
        }

        private enterprisePosition: EnterprisePositionDto;
        set EnterprisePosition(value) {
            this.enterprisePosition = value;
        }
        get EnterprisePosition() {
            return this.enterprisePosition;
        }

        private permissions: Array<models.PermissionDto>;
        get Permissions() {
            return this.permissions;
        }
        set Permissions(value) {
            this.permissions = value;
        }
    }
}

File: Kms.Security.Common/DataContract/EnterprisePositionPostDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Kms.Security.Common.Domain
{
    public class EnterprisePositionPostDto 
    {
        [DataMember]
        public Guid Id {  set; get; }
        [DataMember]
        public string Title { get;  set; }
        [DataMember]
        public EnterprisePositionDto EnterprisePosition { get;  set; }
        [DataMember]
        public Guid EnterprisePositionId { get;  set; }
        [DataMember]
        public string Description { get;  set; }

        [DataMember]
        public IList<PermissionDto> Permissions{ get; set; }

    }
}
